backend/http_exception.py

```diff
@@ -0,0 +1,24 @@
+from typing import Dict, Optional, Tuple
+
+
+class HTTPException(Exception):
```

Comment:
I think for this case one exception for all errors might be enough, but more common way is to implement a tree of exceptions for different kinds of errors. For example, `ResourceExistsException` (code=409) that derived from `HTTPException` and might be raised when some resource already exists. Then you can inherit new type of exception from `ResourceExistsException`, 'UserExistsException' (also with code=409 and maybe different error message) for example. And idea is to cover all edge situations with exceptions that might be serialized to json with needed http code